### PR TITLE
Changing the Date of First Payment should not change the Opportunity Close Date

### PR DESCRIPTION
**Pull request by dfuller114**
Added the setDefaults function to the `giftPaymentSchedulerHelper` so that the payment scheduler has its own default function decoupled from the gift entry form. This prevents the close `Opportunity Close Date` in the scheduler from having a two-way binding with the `First Payment Date` in the gift entry form.

# Critical Changes

# Changes
In the Single Gift Entry form, the `Opportunity Close Date` and the `First Payment Date` will no longer auto-update one another, keeping both values in sync. However, both fields will continue defaulting to today's today when the Single Gift Entry form initially loads.

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
Open the `Single Gift Entry` form. The `Opportunity Close Date` and `First Payment Date` should initially default to today's date. When either of the fields are changed after that, the other field should not auto-update to the same value.

### CODE REVIEW

**Comment by dfuller114** [comment]
**lurch: attach W-037246

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001Ho9rQAC" target="_blank">W-037246</a>

**Comment by dfuller114** [comment]
This PR is currently holding for https://github.com/SalesforceFoundation/gem/pull/253 to be approved and merged so that the test classes are fixed.

File: src/aura/giftEntryForm/giftEntryFormHelper.js [hunk 1]
```diff
@@ -156,7 +156,7 @@
             component.set(vField, val);
         }
     },
-    setDefaults: function(component, opp) {
+    setDefaults: function(component) {
```
Comment (bdvorachek) [inline src/aura/giftEntryForm/giftEntryFormHelper.js]: I see the opp arg is unused, but there's a call that passes that argument on line 102. Can you update that call to not pass in the extra argument too?

Comment (dfuller114) [reply src/aura/giftEntryForm/giftEntryFormHelper.js]: will do

Comment (dfuller114) [reply src/aura/giftEntryForm/giftEntryFormHelper.js]: @bdvorachek just added that change.

**Comment by vmartinov-sforg** [comment]
@ameyersalesforce Do we want to keep this pull request / branch in the GEM repo?

**Comment by vmartinov-sforg** [comment]
Closing stale PRs.

**Pull request closed without merging**